packages/frontend/src/components/common/Page/PageHeader.tsx

```diff
@@ -3,24 +3,22 @@ import { FC } from 'react';
 
 export const PAGE_HEADER_HEIGHT = 80;
 
-const PageHeader: FC = ({ children }) => {
-    return (
-        <Card
-            component={Flex}
-            justify="flex-end"
-            align="center"
-            pos="relative"
-            h={PAGE_HEADER_HEIGHT}
-            px="lg"
-            py="md"
-            bg="white"
-            shadow="xs"
-            radius="unset"
-            sx={{ zIndex: 1 }}
-        >
-            {children}
-        </Card>
-    );
-};
+const PageHeader: FC = ({ children }) => (
+    <Card
+        component={Flex}
+        justify="flex-end"
+        align="center"
+        pos="relative"
+        h={PAGE_HEADER_HEIGHT}
+        px="lg"
+        py="md"
+        bg="white"
+        shadow="0 0 0 1px #bec1c426"
```

Comment:
This is a highly customized shadow. Can you leave a FIXME note to indicate that this shadow should be sourced from the theme config? As of now, we haven't modified the shadow configuration in the Mantine theme and are using the default Mantine settings

Reply:
100%, you're right... I'll add a FIXME yes 👍